notebooks/MAST/TESS/beginner_how_to_use_ffi/beginner_how_to_use_ffi.ipynb

```diff
@@ -0,0 +1,221 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reading the WCS and Calibrated Image\n",
+    "\n",
+    "Now that we have the file, let's store the world coordinate system information for use later.  We can use the astropy.wcs `WCS` function to store the information from the image extension HDU's header.\n",
+    "\n",
+    "The following command opens the file, extracts the WCS and Image data and then closes the file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with fits.open(fits_file, mode = \"readonly\") as hdulist:\n",
+    "    wcs_info = WCS(hdulist[1].header)\n",
+    "    cal_image = hdulist[1].data\n",
+    "    header = hdulist[1].header"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use the header to determine the mid-point of the exposure time for this FFI.\n",
+    "mid_time = (header['TSTOP'] + header['TSTART']) / 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Display the Image\n",
+    "We show the 2048x2048 image, adjusting the scale so that we can see more of the stars.\n",
+    "We also imprint the WCS ontop of the image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize = (12,12))\n",
+    "\n",
+    "plt.subplot(111, projection = wcs_info)\n",
+    "plt.imshow(cal_image, vmin = np.percentile(cal_image,4),vmax = np.percentile(cal_image, 99),origin = \"lower\")\n",
+    "plt.xlabel('RA')\n",
+    "plt.ylabel('Dec')\n",
+    "plt.title(\"TESS Calibrated FFI for Sector 1, Camera 4, CCD 2, Timestamp %f BTJD\" % mid_time)\n",
+    "\n",
+    "cal_image.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## A Few Image Details\n",
+    "We describe some of the details you see on this FFI. For more detailed information please see the TESS Instrument Handbook (Section 4.1).  Notice that our image is (2078, 2136) in size. Calibration row and columns are read-out in addition to the image data. These rows and columns remain in the calibrated image.\n",
+    "\n",
+    "### Leading and Trailing Black\n",
+    "There are 4 sets of 11 columns (44 columns in total) that make up the leading and trailing black.  Each CCD is read in 4 slices and each slice has an 11 column leading black and an 11 column trailing black. All of the leading black are placed in the first 44 columns while the trailing black are the last 44 columns.\n",
+    "\n",
+    "### Virtual, Smear and Buffer\n",
+    "The top 30 rows are, from the top, 10 virtual rows, 10 smear rows and 10 buffer rows.\n",
+    "\n",
+    "### Bright Spot\n",
+    "Data annomalies are discussed in the TESS Setor 1 Data Release Notes. Likely this bright spot is caused by some scattered light."
```

Comment:
I can't tell what spot you mean by "this bright spot," maybe it is clear to scientists? Otherwise maybe a little more description so people can identify it.